src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -97,8 +94,16 @@
   /**
    * Identifies {@link SonarRunnerInstallation} to be used.
    * @since 2.0
+   * @deprecated since 2.5
+   */
+  @Deprecated
+  private transient String sonarRunnerName;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the "transient" modifier from this field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2065)
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)